casper/src/test/scala/coop/rchain/casper/util/rholang/InterpreterUtilTest.scala

```diff
@@ -278,6 +278,183 @@ class InterpreterUtilTest
     b3PostState.contains("@{6}!(6)") should be(true)
   }
 
+
+  val registry =
+  """
+    |new ri(`rho:registry:insertArbitrary`) in {
+    |  new X, Y in {
+    |    ri!(*X, *Y)
+    |  }
+    |}
+  """.stripMargin
+
+  val other =
```

Comment:
This looks like `tut-hello.rho`. Why not read it from this file?

Reply:
would introduce dependency on another module.